requirements.txt

```diff
@@ -2,3 +2,4 @@ pygments>=2.7.4 # not directly required, pinned by Snyk to avoid a vulnerability
 pytz
 requests>=2.0.0
 PyJWT>=2.0.0, <3.0.0
+black
```

Comment:
Prefer to move this to the `tests/requirements` file since it's not a runtime dependency.